docs/installation/hetzner-cloud.md

```diff
@@ -0,0 +1,129 @@
+# Install yaydoc on a server
+In this case we will be using **Hetzner Cloud**.
+
+## Setup
+You can create an account to Hetzner Cloud on their [website](https://www.hetzner.com). After logging in you may need to perform verification in the form of PayPal or Government ID.
+
+### Create a new Project and Connect
+In the [dashboard](https://console.hetzner.cloud/) select 'Add a new project'. Fill all the required fields and proceed to the Project page.
+
+Now select the red 'Add Server' button in the top right. You can choose your desired specifications and location.
+
+After creation, you should have a root password sent to the email address provided. Using this we can connect to our server.
+
+Open up a terminal window (Mac, Linux) or use SSH software (Windows) to connect to the server.
+
+On Linux or Mac systems it is very simple. Substitute ip address for the ip address of your server.
+```
+ssh <ip address> -l root
+```
+Now enter your root password. After connection it will ask you to change the password. You can change it to whatever you desire.
+
+## Installing prerequisites
+Now to install and setup Yaydoc.
+We need to install
+- git
+- npm & nodejs
+- python/python3
+- pip
+
```

Comment:
Please add documentation and commands, what the user has to do exactly.

Reply:
Added!